src/Eshopworld.WorkerProcess/CosmosDistributedLock/DistributedLockNotAcquiredException.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace EShopworld.WorkerProcess.CosmosDistributedLock
+{
+    public class DistributedLockNotAcquiredException : Exception
+    {
+        public DistributedLockNotAcquiredException(string lockName, Exception innerException)
+            : base($"Distributed Lock for document with id: '{lockName}' could not be acquired.", innerException)
```

Comment:
DO you think it makes sense to also give the collection name as lock name is limited to that context only?

Reply:
Yes, I'll add collection name to the message.